index.md

```diff
@@ -16,98 +16,91 @@ order: 1
 ## Introduction
 {:#introduction}
 
-Web accessibility evaluation tools are software programs or online services that help determine if a Web site is accessible. This document highlights features of evaluation tools, and discusses different aspects to consider when selecting these tools. Links to different tools are provided in the [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/).
+Web accessibility evaluation tools are software programs or online services that help determine if a website is accessible. This document highlights features of evaluation tools, and discusses different aspects to consider when selecting these tools. Links to different tools are provided in the [Web Accessibility Evaluation Tools List](https://www.w3.org/WAI/ER/tools/).
 
 ## What to Expect from Evaluations Tools
 {:#Expect}
 
-Web accessibility evaluation tools can help you quickly identify potential accessibility issues. You can use them through all phases of the web design and development process. Tools can provide fully-automated checks, as well as help you with manual review.
+Web accessibility evaluation tools can help you quickly identify potential accessibility issues. You can use them through all phases of the web design and development process. Tools provide fully-automated checks, as well as help you with manual review.
 
 But we cannot check all accessibility aspects automatically. Human judgement is often required. Sometimes evaluation tools can produce false or misleading results. Web accessibility evaluation tools can not *determine* accessibility, they can only *assist* in doing so.
 
 ## Features of Evaluation Tools
 {:#Features}
 
-Web accessibility evaluation tools target different audiences. This includes designers, developers, non-technical content authors, quality assurance testers, and sometimes also end-users. Tools tend to offer different features and functionality which may help users to compare and assess Web accessibility evaluation tools for their specific needs.
+Web accessibility evaluation tools target different audiences. This includes designers, developers, non-technical content authors, quality assurance testers, and sometimes also end-users. Tools tend to offer different features and functionality which may help users to compare and assess web accessibility evaluation tools for their specific needs.
 
-The following features describe tools found on the [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/). 
```

Comment:
The addition of a heading was made very specifically as a response to previous comments to differentiate the features that are filterable from those that are not. I feel strongly this should remain as is.

Reply:
I think this is creating unnecessary complexity. IMO there is a difference between aligning this document with the tools list, and making it a user manual for the tools list.

Also note that some of the "no filterable features" are actually just sub-types of the filterable features, so this separation seems confusing to me.

Reply:
I think we want it to work well for both of these use cases:
1. I was looking at the tools list and did't understand some of the Filters, so I go over to this document for an explanation.
2. I want to learn about Eval Tools, but I'm not even going to look at the Tools List ('cause that's someone else's job).

I think we're moving positively in that direction, but it stills needs some work to get there. Having the items that are Filters separate helps find them; however, the headings is the last version were confusing in a stand-alone document. I think with understanding the multiple use cases, we can find a smooth way to address it. :-)

Reply:
It may be something different than under separate heading, e.g., possibly bold or some other way to distinguish them? /me is *not* suggestion that, just trying to help think outside the box :-)